assets/css/style.css

```diff
@@ -0,0 +1,136 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #d9dcd6;
+}
+
+/*This is the class selector for class name of header */
+.header {
+  padding: 20px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+    "Lucida Sans", Arial, sans-serif;
+  background-color: #2a607c;
+}
+
+.logo {
+  display: inline-block;
+  font-size: 48px;
+  color: #ffffff;
+}
+
+.seo {
+  color: #d9dcd6;
+}
+
+nav {
+  padding-top: 15px;
+  margin-right: 20px;
+  float: right;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  font-size: 20px;
+}
+
+nav ul {
+  list-style-type: none;
+}
+
+nav li {
+  display: inline-block;
+  margin-left: 25px;
+}
+
+nav a {
+  color: #ffffff;
+  text-decoration: none;
+}
+
+/*This is the background image*/
+.banner {
+  height: 800px;
+  width: 100%;
+  margin-bottom: 25px;
+  background-image: url("../images/digital-marketing-meeting.jpg");
+  background-size: cover;
+  background-position: center;
+}
```

Comment:
Whilst adding the image as a background via the CSS does work, unfortunately you can't add an `alt` tag to it which was a requirement of this homework. Consider putting this back into an `img` tag on the index.html